tests/Tests/Unit/RequestBuilderTest.php

```diff
@@ -378,6 +382,20 @@ public function testMethodWithOneOfInQueryString()
         $this->assertSame('some-value', $query['field1']);
     }
 
+    public function testMethodWithNumericEnumsQueryParam()
+    {
+        $message = (new MockRequestBody())
+            ->setName('')
+            ->setNumber(0);
```

Comment:
I'm confused by your test example because neither of these seem like enums. Shouldn't we be testing a request or response message with enums?

I'm thinking that it's because this is specifically for the request, and not the response. If that's the case, we probably can drop the `name` and `number` logic, to make this clear.

Reply:
I think I see your point: The fields on this request message aren't actually part of the test, so let's exclude them. That makes sense to me. Done.